cmd/formatter/colors.go

```diff
@@ -73,10 +73,11 @@ var monochrome = func(s string) string {
 }
 
 func ansiColor(code, s string) string {
-	return fmt.Sprintf("%s%s%s", ansi(code), s, ansi("0"))
+	return fmt.Sprintf("%s%s%s", ansiColorCode(code), s, ansiColorCode("0"))
 }
 
-func ansi(code string) string {
+// Everything about ansiColorCode color https://hyperskill.org/learn/step/18193
+func ansiColorCode(code string) string {
 	return fmt.Sprintf("\033[%sm", code)
 }
```

Comment:
Didn't realize we had this – isn't the stuff in `aec` enough? I think we use it sometimes https://github.com/docker/compose/blob/9b0e3d53eb4191fa031fafe65f74ef377ba5c99d/pkg/progress/colors.go#L30-L36

Reply:
then it's used like https://github.com/docker/compose/blob/9b0e3d53eb4191fa031fafe65f74ef377ba5c99d/pkg/progress/tty.go#L258

Reply:
I need to add the bold to the prefix color :( `\033[36m` -> cyan vs` \033[1;36m` -> bold + cyan